openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
For the record, our CMFD solver also does this (uses scatter-1 vice nu-scatter-1), but it should be of no consequence there since in the end you force the currents to match anyways.